test/test_common/environment.cc

```diff
@@ -148,6 +148,8 @@ bool TestEnvironment::shouldRunTestForIpVersion(Network::Address::IpVersion type
 
 std::vector<Network::Address::IpVersion> TestEnvironment::getIpVersionsForTest() {
   std::vector<Network::Address::IpVersion> parameters;
+  parameters.push_back(Network::Address::IpVersion::v4);
```

Comment:
Is this intended? Revert this file?

Reply:
Sorry, that was an artifact of trying to run coverage locally, reverted.